build/Taskfile

```diff
@@ -11,6 +11,11 @@ task:run() {
     docker:compose up "$@"
 }
 
+task:restart() {
```

Comment:
I think we should add this command to the documentation (`03_commands.md`) to have a summary of all available commands.

Reply:
Good idea, I will add this documentation in my next PR.